packages/components/organisms/f-checkout/src/tenants/en-GB.js

```diff
@@ -39,7 +39,40 @@ const messages = {
 
     allergyText: 'If you or someone you’re ordering for has a food allergy or intolerance, click here',
 
-    switchUserText: 'Not {name}? Click here.'
+    guest: {
+        firstName: 'First name',
+        lastName: 'Last name',
+        email: 'Email'
+    },
+
+    header: {
+        user: {
+            title: '{name}, confirm your details',
+            switchUser: 'Not {name}? Click here.'
+        },
+        guest: {
+            loginTitle: 'How do you want to continue?',
+            loginButton: 'Log in or sign up',
+            option: 'or',
+            guestTitle: 'Checkout as a guest',
+            confirmation: 'Please confirm your delivery details'
+        }
+    },
+
+    termsAndConditions: {
+        text1: 'By placing an order you agree to our ',
+        text2: '. Please read our Privacy Policy ',
+        text3: ' and ',
+
+        termsAndConditionsLinkText: 'Terms and Conditions',
+        termsAndConditionsLinkUrl: 'https://www.just-eat.co.uk/termsandconditions',
+
+        privacyPolicyLinkText: 'Privacy Policy',
+        privacyPolicyLinkUrl: 'https://www.just-eat.co.uk/privacypolicy',
+
+        cookiePolicyLinkText: 'Cookie Policy',
+        cookiePolicyLinkUrl: 'https://www.just-eat.co.uk/cookies-policy'
+    }
```

Comment:
You won't need so many translations if you use `<i18n>` as per my other comment 🙂